### PR TITLE
feat(core): use platform-aware hotkeys

### PR DESCRIPTION
**Pull request by rexxars**
### Description

Introduces a new `<Hotkeys />` component in the `sanity` module which automatically changes `alt` to `option` and vice versa, but otherwise just reuses what's in `@sanity/ui`. For more context, see https://github.com/sanity-io/ui/pull/1168

### What to review

- Shows correct hotkey for publishing and similar on apple devices

### Notes for release

- Hotkeys are now device aware, showing `Option` instead of `Alt` on Apple devices

### CODE REVIEW

File: packages/sanity/src/core/components/Hotkeys.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ *
+ * @public
+ */
+export type HotkeysProps = UIHotkeysProps & {
+  /**
+   * Whether to make the keys platform-aware (eg `alt` to `option` on Apple devices).
+   *
+   * @defaultValue true
+   */
+  makePlatformAware?: boolean
```
Comment (bjoerge) [inline packages/sanity/src/core/components/Hotkeys.tsx:15]: out of curiosity and lack of imagination: in what cases would you want to make it not platform aware?

**Review by bjoerge — COMMENTED** [review]
Very nice, thank you! <3

**Review by bjoerge — APPROVED** [review]

**Review by binoy14 — APPROVED** [review]
LGTM

File: packages/sanity/src/core/components/Hotkeys.tsx (hunk 1 above)
Comment (rexxars) [reply packages/sanity/src/core/components/Hotkeys.tsx:15]: Potentially if showing both alternatives, eg something like:
```
Windows: Ctrl+Alt+I
Mac: Ctrl+Opt+I
```

**Pull request merged**